test/units.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
```

Comment:
do you need this file stored in git at all? You will have test task while the build is creating, so it will add all the files you need.